chimney/src/test/scala/io/scalaland/chimney/PatcherSpec.scala

```diff
@@ -23,12 +23,23 @@ class PatcherSpec extends WordSpec with MustMatchers {
 
       import TestDomain._
 
-      val user = User(10, Email("abc@def.com"), Phone(1234567890L))
       val update = UpdateDetails("xyz@def.com", 123123123L)
 
-      user.patchWith(update) mustBe
+      exampleUser.patchWith(update) mustBe
         User(10, Email("xyz@def.com"), Phone(123123123L))
     }
+
+    "support optional types in patch" in {
```

Comment:
does it support both optional and regular types? i.e. if I had such test:
```
"support mixed optional and regular types" in {
    import TestDomain._

    case class UserPatch(email: String, phone: Option[Phone])
    val update = UserPatch(email = "updated@example.com", phone = None)
    
    exampleUser.patchWith(update) mustBe User(10, Email("updated@example.com"), Phone(1234567890L))
}
```

Reply:
Yes, I expect this test should pass. I will include it to the PR later.

Reply:
Test added. It works without problems, as expected.